### DISCUSSION TITLE
Use Cmd-C, Cmd-V again in oh-my-zsh-equipped Terminal in macOS Sequoia

### DISCUSSION THREAD

**Discussion by murrayE** [ohmyzsh/ohmyzsh#12746] [Help]
Before either upgrading to macOS Sequois or updating to the latest omz, in the macOS Terminal app, I used to be able to select text appearing anywhere on a line in Terminal, copy it to the clipboard with Cmd-C, and paste it into the currently prompted line in Terminal with Cmd-V. 

Now it requires Shift-Cmd-C and Shift-Cmd-V. (Cmd-C without Shift now is not copying to the System clipboard, so of course Cmd-V  without Shift copies the last entry on the system clipboard, not what I just highlighted inside Terminal and tried to copy with Cmd-C).

Is there some way to restore the former behavior?

Note that Cmd-C and Cmd-V work just find in other macOS apps, including TextEdit, BBedit, etc.

**Comment by KianAnbarestani** [comment]
@murrayE 
You can restore the old behavior by customizing Terminal's key bindings. Open Terminal > Preferences > Profiles > Keyboard and modify the key mapping so that Cmd-C and Cmd-V perform the Copy/Paste actions without needing Shift. If necessary, reset to defaults and then reassign the shortcuts you prefer. This should re-enable copying text from anywhere in the line directly with Cmd-C.

**Comment by murrayE** [comment]
In current omz under macOS Sequoia and default zshell, Terminal > Settings [not "Preferences"] > Keyboard displays a list with 2 columns.

The left-hand column displays keys, the right column displays "Actions" such as \033[29~ that are a complete mystery to me.

> On Apr 16, 2025, at 9:33 AM, Kian Anbarestani ***@***.***> wrote:
> 
> 
> ### @murrayE <https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FmurrayE&data=05%7C02%7Cmurray%40cas.umass.edu%7Ce928baa659824f4d30e408dd7ceb4f48%7C7bd08b0b33954dc194bbd0b2e56a497f%7C0%7C0%7C638804072315952339%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=%2FWg3qY53kEwxG2nlYutjHLJbY9%2FwbluRMqpsjnlWtw4%3D&reserved=0>
> You can restore the old behavior by customizing Terminal's key bindings. Open Terminal > Preferences > Profiles > Keyboard and modify the key mapping so that Cmd-C and Cmd-V perform the Copy/Paste actions without needing Shift. If necessary, reset to defaults and then reassign the shortcuts you prefer. This should re-enable copying text from anywhere in the line directly with Cmd-C.
> 
> —
> Reply to this email directly, view it on GitHub <https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fohmyzsh%2Fohmyzsh%2Fdiscussions%2F12746%23discussioncomment-12855062&data=05%7C02%7Cmurray%40cas.umass.edu%7Ce928baa659824f4d30e408dd7ceb4f48%7C7bd08b0b33954dc194bbd0b2e56a497f%7C0%7C0%7C638804072315987003%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=l4maSvy2sAR5vAcZ3r6BfoUbJ%2BxR02wJmxC157n71qs%3D&reserved=0>, or unsubscribe <https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABILR2LTW455DN6TJXE2X532ZZL3PAVCNFSM6AAAAABP2XXNQKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOBVGUYDMMQ&data=05%7C02%7Cmurray%40cas.umass.edu%7Ce928baa659824f4d30e408dd7ceb4f48%7C7bd08b0b33954dc194bbd0b2e56a497f%7C0%7C0%7C638804072316061064%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=9uKKEhWblVsE33YboGo%2B3UdE54as2V056uK2SZ%2BQeOs%3D&reserved=0>.
> You are receiving this because you were mentioned.
> 

——
Murray Eisenberg                                ***@***.***
Mathematics & Statistics Dept.       
University of Massachusetts Amherst
